src/main/resources/application.properties

```diff
@@ -1,7 +1,10 @@
+jwt.secret=HRlELXqpSB
 spring.config.import=file:env.properties
 spring.jpa.hibernate.ddl-auto=update
 spring.datasource.url=jdbc:mysql://localhost:3307/icu-connect?createDatabaseIfNotExist=true&autoReconnect=true
 spring.datasource.username=${DB_USER}
 spring.datasource.password=${DB_PASSWORD}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect
+spring.main.allow-circular-references=true
```

Comment:
Why is this needed?

Reply:
There is a circular dependency that I was not able to solve, so this is a short time fix